### PR TITLE
Changed compile to implementation, updated buildToolsVersion & multidex

### PR DESCRIPTION
**Pull request by suthat**

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -1,9 +1,8 @@
-
 apply plugin: 'com.android.library'
 
 android {
     compileSdkVersion 25
-    buildToolsVersion "25.0.1"
+    buildToolsVersion "28.0.3"
```
Comment (mvayngrib) [inline android/build.gradle]: can you remove `buildToolsVersion` ? I think it's optional these days, and I don't want to force people to upgrade to 28.0.3

File: android/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,8 +1,4 @@
-
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="io.tradle.nfc">
-
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="io.tradle.nfc">
```
Comment (mvayngrib) [inline android/src/main/AndroidManifest.xml:1]: next time please limit the changes to the specific scope of the PR, thanks! This time i'll accept :)

File: android/build.gradle [hunk 3]
```diff
@@ -31,14 +30,14 @@ android {
...
 dependencies {
-    compile 'com.android.support:multidex:1.0.1'
-    compile 'com.android.support:appcompat-v7:25.1.0'
-    compile 'org.jmrtd:jmrtd:0.5.5'
-    compile 'net.sf.scuba:scuba-sc-android:0.0.9'
-    compile 'com.madgag.spongycastle:prov:1.54.0.0'
-    compile 'edu.ucar:jj2000:5.2'
-    compile 'com.github.mhshams:jnbis:1.1.0'
-    compile 'com.facebook.react:react-native:+'
+    implementation 'com.android.support:multidex:1.0.3'
```
Comment (mvayngrib) [inline android/build.gradle:33]: next time please upgrade deps in a separate PR

**Review by mvayngrib — CHANGES_REQUESTED** [review]
please remove `buildToolsVersion` instead of setting `28.0.3`, or default to root project, like here: https://github.com/react-native-community/react-native-camera/blob/master/android/build.gradle#L23

File: android/build.gradle (hunk 1 above)
Comment (suthat) [reply android/build.gradle]: Sure. I Got it.

File: android/src/main/AndroidManifest.xml (hunk 2 above)
Comment (suthat) [reply android/src/main/AndroidManifest.xml:1]: Pardon me.

File: android/build.gradle (hunk 3 above)
Comment (suthat) [reply android/build.gradle:33]: Roger that.

**Review by mvayngrib — APPROVED** [review]

**Comment by mvayngrib** [comment]
@suthat thanks! released in 1.0.3

**Pull request merged**